### DISCUSSION TITLE
error trying to make a change on a submission  https://cloud.walletconnect.com/app/project?uuid=6620aef8-7cd1-4623-958b-a1ed4d4d7431

### DISCUSSION THREAD

**Discussion by neri-ripio** [WalletConnect/walletconnect-monorepo#2506] [Developer Support]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/129459236/937f79eb-508b-428e-aa2c-095988d60312)

**Comment by nad33mahm3d** [comment]
I am also facing the same issue with my submission

**Comment by finessevanes** [comment]
Hey - this issue should have been fixed. @neri-ripio @nad33mahm3d are you still experiencing this issue?
